.github/workflows/ci.yml

```diff
@@ -15,13 +15,14 @@ jobs:
           - shard.yml
         crystal_version:
           - 1.0.0
-          - 1.1.0
-          - 1.2.0
+          - 1.1.1
+          - 1.2.2
+          - 1.3.0
```

Comment:
🙏 Please, sir...can we remove `1.1.1` and `1.2.2` from these check?

Reply:
😂  yeah, this is what I was asking about before. So do we only keep 2 in here plus the nightly? Whatever the lowest version we support, and whatever the highest version is? In this case 1.0.0, and 1.3.0?

Reply:
That's my opinion. I think it's unlikely we'd break anything on the in-between versions and it will cut down on CI time

Reply:
oh actually... for the formatting we only really need to include latest. I'm gonna drop the 1.0 here too